portability-types-client/src/main/java/org/dataportabilityproject/types/client/transfer/DataTransfer.java

```diff
@@ -32,19 +32,25 @@
     }
 
     private Status status;
+    private String nextURL;
 
     @JsonCreator
     public DataTransfer(
             @JsonProperty(value = "source", required = true) String source,
             @JsonProperty(value = "destination", required = true) String destination,
             @JsonProperty(value = "transferDataType", required = true) String transferDataType,
-            @JsonProperty(value = "status", required = true) Status status) {
+            @JsonProperty(value = "status", required = true) Status status,
+            @JsonProperty(value = "nextURL", required = true) String nextURL) {
```

Comment:
Will this be a property only on certain types of requests or all? My guess was that this might not be required.

Can we add documentation as well.

Reply:
Done - I added the comment on the class member since adding it at the constructor looked awkward.

Reply:
I was thinking the DataTransfer entity would be (eventually) POSTed to a URL when the user wanted to transfer data, but before they have supplied credentials. For my own understanding, what does nextURL do?

Reply:
My intention was that nextURL would tell the front end client which urls to redirect to after handling the response. 

There are three main points where the api server needs to inform the front end where to redirect to: 
- in ConfigureHandler (which is wrongly named, should be something like CreateJobHandler), after the front end client creates the job, it should redirect to oauth flow for the source service. 
- in ImportSetupHandler, the client should be redirected to the oauth flow for the destination service 
- in CopySetup, the client should redirect to the CopyHandler which will start the Copy.

I imagine if this is meant to be a sort of status page, it could be used to redirect the user to the correct page to go to. Example: if the user hasn't supplied creds yet, we could use it to redirect to the oauth flows. Or if theyre checking on the status of the copy, we could redirect them back to the in-status page (or a successful page if the copy is complete)? 

Does that make sense?